docs-ref-services/latest/cognitiveservices/bing-custom-image-search-readme.md

```diff
@@ -2,10 +2,9 @@
 title: Bing Custom Image Search 
 author: PatrickFarley
 ms.author: pafarley
+description: Bing Search APIs are moving from Cognitive Services to Bing Search Services.
```

Comment:
isn't the description for this one identical to the Azure/azure-docs-sdk-java? but well, if validation is ok with it, then I guess it's fine. :)

Reply:
Yeah, that was the only information given in the files so...